Source/WebKit/UIProcess/PageClient.h

```diff
@@ -465,6 +465,7 @@ class PageClient : public CanMakeWeakPtr<PageClient> {
     virtual void restorePageCenterAndScale(std::optional<WebCore::FloatPoint> center, double scale) = 0;
 
     virtual void elementDidFocus(const FocusedElementInformation&, bool userIsInteracting, bool blurPreviousNode, OptionSet<WebCore::ActivityState::Flag> activityStateChanges, API::Object* userData) = 0;
+    virtual void presentFindNavigatorWithoutReplace() = 0;
```

Comment:
I think we should rename this method (and the ones below) to something like `updateFindResults`, since we're really working around the fact that UIKit doesn't currently expose a better way to update results, and the find navigator is already visible.